File: christmas/december task 2/function tasks/IsEven.java
import java.util.Scanner;   
     public class IsEven {
        public static void main(String...Odili) {
         
        
            System.out.println(evenNumber(2));


        }

        public static boolean evenNumber(int number){
            Scanner input = new Scanner(System.in);

            System.out.println("Enter any integer");
                 number = input.nextInt();

            if(number % 2 == 0){
                return true;
            }
            
            else{
                return false;
            }

        }


    }

File: christmas/december task 2/function tasks/IsEven.js
const prompt = require('prompt-sync')();
  
function evenNumber(number){
 
   number = prompt("Enter any integer ");
       
    if(number % 2 == 0){
        return true;
    }
    
    else{
        return false;
    }

}
number = 0;

console.log(evenNumber(number));

